src/transformers/models/gptj/modeling_gptj.py

```diff
@@ -207,12 +191,13 @@ def forward(
         key = self._split_heads(key, self.num_attention_heads, self.head_dim, True)
         value = self._split_heads(value, self.num_attention_heads, self.head_dim, False)
 
-        seq_len = key.shape[1]
-        offset = 0
+        embed_positions = self.embed_positions
+        if embed_positions.device != position_ids.device:
+            embed_positions = embed_positions.to(position_ids.device)
+            self.embed_positions = embed_positions
```

Comment:
@gante let me know if there's a better way to handle this

Reply:
LGTM 👍